components/loading-bar.vue

```diff
@@ -0,0 +1,322 @@
+<template>
+  <v-footer absolute color="#292c31" class="font-weight-medium" style="min-width:745px;" ref="footer">
+    <div 
+      class="content-width-100 px-16"
+    >
```

Comment:
From line 3 to line 5 could be one line.
Please refer to the case using multiple lines on https://www.w3schools.com/html/html5_syntax.asp
Or you could make a rule of coding style, and keep it.